### ISSUE TITLE
Add documentation for response headers

### ISSUE THREAD

**Issue by blakev** [channelcat/sanic#178]
Just spent a long time trying to track down a bug related to headers where a key's value was an int.

The header dictionary must be in the format `[str, str]` or the response will silently fail.

```python
    headers = {
        'Age': str(int(time.time() - store_time)),
    }
```

**Comment by wojunnihou** [comment]
可能还会有编码的问题,若能在设置中文的时候，智能识别编码就更好了

**Comment by seemethere** [comment]
Maybe it would be useful to have it try to convert it to a str on it's own?

**Comment by seemethere** [comment]
Closed per #197
